api/swagger.yaml

```diff
@@ -1267,6 +1267,9 @@ definitions:
       Containers:
         x-nullable: false
         type: "integer"
+      Platform:
+        description: "Platform of the image"
+        type: "string"
```

Comment:
Can you add;

- an `example:` key with a value (e.g. "linux/amd64")? 
- extend the description a bit to explain the field is omitted if empty, the information included (`<platform>/<architecture>`) (or is it "operating system"?)
- also update the examples section for the `/images/json` endpoint and add example values there? https://github.com/moby/moby/blob/master/api/swagger.yaml#L5965

Reply:
This also needs a line in https://github.com/moby/moby/blob/master/docs/api/version-history.md, but should go in a new API version; I prepared a PR for that here: https://github.com/moby/moby/pull/34908

That already has two LGTM's so we can merge if this PR gets ok'd

Reply:
OK, so adding an example in the /images/json bit
With the latest update, the description bit goes away as it's now using the OCI image spec.

I'll follow up the version-history.md once that's merged.